Unix/provmgr/provmgr.c

```diff
@@ -190,7 +196,8 @@ static Library* MI_CALL _OpenLibraryInternal(
 
             if (!p->handle)
             {
-                trace_SharedLib_CannotOpenSecondTry(scs(proventry->libraryName), tcs(Shlib_Err()));
+                trace_SharedLib_CannotOpenSecondTry(
+                    scs(proventry->libraryName), tcs(Shlib_Err()));
```

Comment:
looks this doesn't same as previous "proventry->interpreter ? "OMIScriptProvider" :  proventry->libraryName", suggest to new a TChar* var1=proventry->interpreter ? "OMIScriptProvider" :  proventry->libraryName; in the begin of function "static Library* MI_CALL _OpenLibraryInternal("